arbnode/espresso_batcher_addr_monitor.go

```diff
@@ -373,17 +259,14 @@ func (b *BatcherAddrMonitor) backfill(ctx context.Context) error {
 		}
 
 		log.Info("batcher addr monitor backfilling", "lastProcessedHeight", lastProcessedHeight, "latestParentHeight", latestParentHeight, "blocksToRead", blocksToRead)
-		events, err := b.LookupAddressUpdates(ctx, lastProcessedHeight+1, lastProcessedHeight+blocksToRead)
+		updates, err := b.LookupAddressUpdates(ctx, lastProcessedHeight+1, lastProcessedHeight+blocksToRead)
```

Comment:
I dont see the reason why we still need backfilling, why cant it be the case that we start with a list of batch poster addresses in the config. We dont backfill. In case there is a `batcher address` which we havent seen before we just check the contract and see if its valid or not.